src/redux/gas.js

```diff
@@ -116,6 +117,11 @@ export const gasUpdateGasPriceOption = newGasPriceOption => (
     },
     type: GAS_UPDATE_GAS_PRICE_OPTION,
   });
+  const nextSpeedGweiValue = get(
+    gasPrices,
+    `[${newGasPriceOption}].value.display`
+  );
+  analytics.track('Updated Gas Price', { gasPrice: nextSpeedGweiValue });
```

Comment:
I think it's more useful to track the speed chosen  (newGasPriceOption) rather than the actual  value